.air.toml

```diff
@@ -4,7 +4,7 @@ tmp_dir = ".air"
 [build]
 cmd = "make backend"
 bin = "gitea"
-include_ext = ["go", "tmpl"]
+include_ext = ["go", "tmpl", "ini"]
 exclude_dir = ["modules/git/tests", "services/gitdiff/testdata", "modules/avatar/testdata", "models/fixtures", "models/migrations/fixtures", "modules/migration/file_format_testdata", "modules/avatar/identicon/testdata"]
-include_dir = ["cmd", "models", "modules", "options", "routers", "services"]
+include_dir = ["cmd", "models", "modules", "options/locale", "routers", "services"]
```

Comment:
IIRC the `options/locale` should be auto-reloaded in dev mode?

Reply:
And the `ini` has been excluded above, why the `options/locale` should be included here (it only contains ini files)

Reply:
That's not how air works. Included extensions need to be in `include_ext`. It makes no sense to monitor other option dirs because they only contain extensionless files which will never match.

Reply:
But why `options/locale` should appear in the `included_dir`? The dev mode supports auto reloading the locales without restarting.

Reply:
Because it's unnecessary looking into dirs that don't contain any matched files. We can revert this to `options` if you like. It did not work before because of missing `include_ext`.

Reply:
I don't use the air. But I guess some developers may not like the new behavior: any change in locale ini causes restarting (which seems unnecessary)

Reply:
Backend output depends on locale ini files, so they are a natural dependency and it should restart.

Reply:
Really? I am not sure .... 🤣 

zeripath had written a lot of code to do the hot-reloading, to make the debug process not restart. If these code doesn't work, should it be removed?

Reply:
> I don't use the air

Curious, what do you use instead?

Reply:
> > I don't use the air
> 
> Curious, what do you use instead?

* backend: restart manually (just click a toolbar button)
* frontend: make watch-frontend
* tmpl / locale : auto reload in dev mode

Reply:
> tmpl / locale : auto reload (hot reload) in dev mode

Ah, it seems that actually works, so no need for air reload. Wasn't aware of that behaviour.

Reply:
Air change reverted.